manifests/install/repos/extra.pp

```diff
@@ -30,12 +30,7 @@
   }
 
   if $ipa_authentication and $configure_ipa_repo {
-    yumrepo { 'adelton-identity':
```

Comment:
I don't think you should remove the resource, else that isn't really a deprecation, it's an incompatible removal.  It should probably remain until the next major module release.